brave/brave-core
# Storybook: Add Confirm Panel to connect-with-site-component

Resolves <https://github.com/brave/brave-browser/issues/14872>

Added Confirm panel to connect-with-site component and added to storybook

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that the storybook shows new component.
3. Check functionality of the new component.


https://user-images.githubusercontent.com/40611140/112027164-8b655e00-8afc-11eb-9a11-ce6603e62c82.mov
